crates/http/src/routes.rs

```diff
@@ -14,20 +14,29 @@ pub struct Router {
     pub(crate) routes: IndexMap<RoutePattern, String>,
 }
 
+type DuplicateRoute = (RoutePattern, String, String);
```

Comment:
Could you add a comment on how to interpret the two `String`s of this tuple? At glance it's not immediately obvious. Thanks!

Reply:
I knew I was shooting myself (and my readers) in the foot trying to get away with a tuple rather than a struct.  Thank you for keeping me honest!